com.microsoft.mrtk.input/Interactors/Gaze/GazeInteractor.cs

```diff
@@ -8,7 +8,6 @@ namespace Microsoft.MixedReality.Toolkit.Input
 {
     /// <summary>
     /// An XRRayInteractor that enables eye gaze for focus and interaction.
-    /// Functionally empty while we learn more about the tooling necessary for successful gaze interaction development
```

Comment:
Any reason we want to purge this comment? 😅

Reply:
Because it's now _intentionally_ empty, lol. We've definitely learned more about the "tooling necessary for successful gaze interaction" since then

Reply:
If it's intentionally empty now, should we just remove it?

Reply:
I have similar concerns. On one hand, it feels like there's no real need to have "different flavors" of gaze interactor, but that's maybe because we haven't really thought of it yet.

Reply:
I think it's worth having for folks who don't want to take the performance penalty for fuzzy gaze. The fuzzy gaze calculations are one of the most performance intensive parts of MRTK3 right now, so, it is nice to have a vanilla GazeInteractor that works "almost as good" for less perf overhead.